api-docs/client-interactions.md

```diff
@@ -0,0 +1,198 @@
+# Client Interactions
+
+The client interactions document direct encounters with the clients.
+
+Here is the list of the valid interaction types:
+
+- `clientInteractionCreated`,
+- `clientInteractionUpdated`,
+- `clientInteractionDeleted`,
+
+## Get a single Client Interaction
+
+### Request
+
+```http
+GET /api/clients/:id/interactions/:id
+```
+
+### Response
+
+```json
+{
+  "id": 1,
+  "title": "Nutrition Program - SNAP",
+  "interactionType": "walk-in",
+  "notes": "Application successfully filled out. Decision pending.",
```

Comment:
Client logs have a `description` property, not a `notes` property. Maybe rename this to match?

Reply:
Change made